CMakeLists.txt

```diff
@@ -43,8 +42,6 @@ option(SSG_PRODUCT_WEBMIN "If enabled, the Webmin SCAP content will be built" TR
 option(SSG_CENTOS_DERIVATIVES_ENABLED "If enabled, CentOS derivative content will be built from the RHEL content" TRUE)
 option(SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED "If enabled, Scientific Linux derivative content will be built from the RHEL content" TRUE)
 
-option(SSG_COMPATIBILITY_SYMLINK "If enabled a compatibility symlink will be created in the old SCAP Security Guide location - /usr/share/xml/scap/contente (only on a UNIX derived operating system)" FALSE)
-
```

Comment:
@mpreisler why remove this code? If SSG_COMPATIBILITY_SYMLINK is FALSE, it doesn't create the symlink, so then, the code shouldn't have to be removed, right?